plugins/src/lib.rs

```diff
@@ -363,7 +369,7 @@ where
     /// The state gets cloned for each request
     state: S,
     options: Vec<ConfigOption>,
-
+    configuration: Configuration,
```

Comment:
```suggestion
    /// The cln configuration received after the init msg
    configuration: Configuration,
```

Reply:
I commented this key and the "options" key